docs/public/presets/toml/jetpack.toml

```diff
@@ -79,53 +83,61 @@ $openstack\
 $azure\
 $crystal\
 $custom\
-$jobs\
 $status\
 $os\
-$container\
 $battery\
-$time\
-"""
+$time"""
 
-add_newline = true
+[fill]
+symbol = ' '
 
 [character]
 format = "$symbol "
-success_symbol = "[◉](bold italic bright-yellow)"
-error_symbol = "[⊘](italic purple)"
-
-[env_var.VIMSHELL] # vim subshell
+success_symbol = "[◎](bold italic bright-yellow)"
+error_symbol = "[○](blink italic purple)"
+# vicmd_symbol = "[■](italic dimmed green)"
+# not supported in zsh
+# vimcmd_replace_one_symbol = "◌"
+# vimcmd_replace_symbol = "□"
+# vimcmd_visual_symbol = "▼"
```

Comment:
Why are theses commented?

Reply:
Taplo was not happy. I'll remove these lines.

Reply:
I think it needs to be `vimcmd_symbol` (it was renamed and taplo doesn't know about the `vicmd_symbol` alias).

Reply:
Oh wow! I couldn't see the difference to save my life. That `m` was invisible to me. Thanks for pointing it out! :bow: